netbox/templates/virtualization/virtualmachine.html

```diff
@@ -325,5 +328,21 @@ <h1>{% block title %}{{ virtualmachine }}{% endblock %}</h1>
     $(this).children('span').toggleClass('glyphicon-check glyphicon-unchecked');
     return false;
 });
+$('input.interface-filter').on('input', function() {
```

Comment:
Can you move this code somewhere so that it can be referenced by either template without being repeated? Maybe `templates/dcim/inc/interface_toggles.js`? (The "toggle IP" bit above it can be moved too.) I just know that if we keep two separate copies of the code we'll inevitably forget to update one at some point.

Reply:
For the sake of performance (caching), I'm gonna put it as a static `.js` file.